File: JAVA PROGRAMMING/Test44.java
// 44. NTH TERM OF FIBONACCI SERIES
import java.util.Scanner;
public class Test {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter Range Value: ");
        int range = in.nextInt();

        int num0 = 0, num1 = 1, temp = 0;
        for(int i = 3; i <= range; i++) {
            temp = num0 + num1;
            num0 = num1;
            num1 = temp;
        }
        System.out.println(range+"th Value Of Fibonacci Series Is: "+temp);
        in.close();
    }
}

File: TYPESCRIPT PROGRAMMING/Test44.ts
// 44. NTH TERM OF FIBONACCI SERIES
const nthfibonacci = (range: number): void => {
    let num0: number = 0;
    let num1: number = 1;
    let temp: number = 0;
    for(let i: number = 3; i <= range; i++) {
        temp = num0 + num1;
        num0 = num1;
        num1 = temp;
    }
    console.log(`${range}th Fibonacci Series Is: ${temp} `);
}
nthfibonacci(10);
nthfibonacci(20);